appium/tests/screenobjects/create-key.screen.ts

```diff
@@ -31,6 +31,16 @@ class CreateKeyScreen extends BaseScreen {
     }
 
     setPassPhrase(text: string = CommonData.account.passPhrase) {
+        // retrying several times because following login, we switch
+        //   from webview to our own view and then to another one several
+        //   times, which was causing flaky tests. Originally we did a 10s
+        //   delay but now instead we're retrying once per second until
+        //   we see what we expect.
+        let count = 0;
+        do {
+            browser.pause(1000);
+            count++;
+        } while(this.enterPassPhraseField.isDisplayed() !== true || count <= 15);
```

Comment:
I don't think this does what you think it does. It looks to me it will try 15 times regardless if it showed before or not. Have you tested how many iterations it does?

Reply:
@tomholubno, we have "||", it is checking one of arguments is true, if the filed is displayed on 5th iteration, we break from loop, but if the field is not displayed after 15 iterations we finish loop and go to next step
I will check how many iterations appear

Reply:
@tomholub sorry, You was right, I changed || to &&

Reply:
@tomholub checked iterations count
we have max 3 iterations
<img width="1175" alt="Screenshot 2021-11-12 at 11 02 20" src="https://user-images.githubusercontent.com/91205853/141440353-efd8e13c-686b-45aa-8529-04d6eb7a35ab.png">

Reply:
That looks better